sphinx_gherkindoc/cli.py

```diff
@@ -80,6 +80,11 @@ def process_args(
             toc_filename = output_path / make_flat_name(current.path_list, is_dir=True)
         toc_file.write_to_file(toc_filename)
 
+        if args.pytest_parser:
+            from .pytest_bdd_writer import feature_to_rst
+        else:
+            from .behave_writer import feature_to_rst  # type: ignore
```

Comment:
Sadly from a historical perspective, behave is the default if nothing else is specified. Much as I would like to require the parser (well, the syntax, technically) to be actively declared in all cases, that would break existing clients.
Unless we go to a full-on major version number change. :-)